sutraani/data.txt

```diff
@@ -77,7 +77,7 @@
             "sskn": "0",
             "plskn": "0",
             "lpn": "0",
-            "pc": "न$S$0$0$##धातुलोपे$S$7$1$##आर्धधातुके$S$7$1$",
+            "pc": "न$S$0$0$##धातुलोप$S$7$1$##आर्धधातुके$S$7$1$",
```

Comment:
Please revert this, the correct word here is धातुलोपे.

Reply:
Sorry 
Created a new pull request regarding this